### PR TITLE
Updating Twitch Articles regarding YouTube switchover

### LINKED ISSUE #741: Append and Remove Twitch Articles
This is due to the recent switchover to using YouTube for livestreams and summits.

### PR DESCRIPTION
**Pull request by theflametrooper**
Changed:

- [x] Twitch.md
- [x] Creating-Twitch-Account.md
- [x] Stream-Your-Coding-Sessions-on-Twitch.tv.md

closes #741

### CODE REVIEW

**Comment by Rafase282** [comment]
I think it is a good idea to not just delete them, it will help with transition.

**Comment by abhisekp** [comment]
@theflametrooper As @Rafase282 says, we may keep the existing one as

> ### Twitch is a popular live streaming service that Free Code Camp moved away from

> As much as we enjoyed using this service, streams were few and far between and therefore it wasn't serving it's purpose.

> We have decided to move our focus onto using YouTube(https://gitter.im/FreeCodeCamp/YouTube) for broadcasting our monthly summits and pair programming as well as uploading tutorials and hikes.

> You can find the old twitch channel here: https://www.twitch.tv/freecodecamp You can find our YouTube channel here: https://www.youtube.com/freecodecamp

> ----

> ----

> ----
~~FreeCodeCamp uses Twitch.tv to broadcast live streams. This can be anything from live pair programming to our monthly Summit, where big news events and new developments are announced.~~

> ~~All levels of coders are encouraged to stream on the FreeCodeCamp twitch channel. If this is something that would interest you, please contact the twitch mods on gitter.~~

> ~~More information on how to stream can be found here.~~

> ~~You can find the channel here: http://www.twitch.tv/freecodecamp~~

File: Creating-Twitch-Account.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ On this page you can change your profile picture, email address and write a bio.
 
 ## Following FreeCodeCamp
 
+> We are currently moving away from using Twitch, we will update this page once details have been finalised.
```
Comment (alayek) [inline Creating-Twitch-Account.md]: I know that *finalised* and _finalized_ both are correct, but *finalized* is the more common one, at least as far as English in US goes. It's the same _colour_ vs. *color* thing. Maybe you should use ~~finalised~~ **finalized**.

Comment (theflametrooper) [inline Creating-Twitch-Account.md]: Sorry, my fault! (I'm from the UK)

**Comment by alayek** [comment]
# LGTM :+1: 

Fix the one issue I mentioned in line note, and we are good to go.

File: Twitch.md [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+You can find our YouTube channel here: https://www.youtube.com/freecodecamp
+
+___________________________
+___________________________
+
+~~FreeCodeCamp uses Twitch.tv to broadcast live streams. This can be anything from live pair programming to our monthly Summit, where big news events and new developments are announced.~~
+
+~~All levels of coders are encouraged to stream on the FreeCodeCamp twitch channel. If this is something that would interest you, please contact the twitch mods on gitter.~~
+
+~~More information on how to stream can be found here.~~
```
Comment (abhisekp) [inline Twitch.md]: This line and the next line doesn't have to be ~~DELETED~~.

File: Creating-Twitch-Account.md [hunk 3]
```diff
@@ -26,6 +26,8 @@ On this page you can change your profile picture, email address and write a bio.
 
 ## Following FreeCodeCamp
 
+> We are currently moving away from using Twitch, we will update this page once details have been finalized.
```
Comment (abhisekp) [inline Creating-Twitch-Account.md]: Make this more visible.

File: Stream-Your-Coding-Sessions-on-Twitch.tv.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
+> We are currently moving away from using Twitch, we will update this page once details have been finalized.
```
Comment (abhisekp) [inline Stream-Your-Coding-Sessions-on-Twitch.tv.md]: Also make this more visible.

File: Twitch.md (hunk 2 above)
Comment (theflametrooper) [inline Twitch.md]: Oh that's what you suggested in your other comment.

File: Twitch.md [hunk 5]
```diff
@@ -1,7 +1,18 @@
...
+You can find our YouTube channel here: https://www.youtube.com/freecodecamp
+
+___________________________
+___________________________
+
+~~FreeCodeCamp uses Twitch.tv to broadcast live streams. This can be anything from live pair programming to our monthly Summit, where big news events and new developments are announced.~~
+
+~~All levels of coders are encouraged to stream on the FreeCodeCamp twitch channel. If this is something that would interest you, please contact the twitch mods on gitter.~~
+
+You can find the channel here: (http://www.twitch.tv/freecodecamp)
```
Comment (abhisekp) [inline Twitch.md]: you may remove this line as it was mentioned above

File: Twitch.md [hunk 6]
```diff
@@ -1,7 +1,18 @@
...
+As much as we enjoyed using this service, streams were few and far between and therefore it wasn't serving it's purpose.
 
-More information on how to stream can be found [here](Stream-Your-Coding-Sessions-on-Twitch.tv).
+We have decided to move our focus onto using YouTube(https://gitter.im/FreeCodeCamp/YouTube) for broadcasting our monthly summits and pair programming as well as uploading tutorials and hikes.
 
-You can find the channel here: http://www.twitch.tv/freecodecamp
\ No newline at end of file
+You can find the old twitch channel here: https://www.twitch.tv/freecodecamp
+
+You can find our YouTube channel here: https://www.youtube.com/freecodecamp
```
Comment (abhisekp) [inline Twitch.md]: YouTube chanel before **_old_** Twitch channel

File: Stream-Your-Coding-Sessions-on-Twitch.tv.md [hunk 7]
```diff
@@ -1,3 +1,5 @@
+> #### We are currently moving away from using Twitch, we will update this page once details have been finalized.
```
Comment (abhisekp) [inline Stream-Your-Coding-Sessions-on-Twitch.tv.md]: This doesn't appear as it should. Remove the quote.

File: Creating-Twitch-Account.md [hunk 8]
```diff
@@ -26,6 +26,8 @@ On this page you can change your profile picture, email address and write a bio.
 
 ## Following FreeCodeCamp
 
+>  #### We are currently moving away from using Twitch, we will update this page once details have been finalized.
```
Comment (abhisekp) [inline Creating-Twitch-Account.md]: Remove the blockquote here too. As it doesn't appear as it should.

Comment (abhisekp) [inline Creating-Twitch-Account.md]: See https://github.com/theflametrooper/wiki/blob/fix/twitchtoyoutube/Creating-Twitch-Account.md#following-freecodecamp

File: Twitch.md [hunk 9]
```diff
@@ -1,7 +1,17 @@
-FreeCodeCamp uses Twitch.tv to broadcast live streams. This can be anything from live pair programming to our monthly Summit, where big news events and new developments are announced.
+### Twitch is a popular live streaming service that Free Code Camp moved away from
 
-All levels of coders are encouraged to stream on the FreeCodeCamp twitch channel. If this is something that would interest you, please contact the twitch mods on [gitter](https://gitter.im/FreeCodeCamp/LiveCoding).
+As much as we enjoyed using this service, streams were few and far between and therefore it wasn't serving it's purpose.
 
-More information on how to stream can be found [here](Stream-Your-Coding-Sessions-on-Twitch.tv).
+We have decided to move our focus onto using YouTube(https://gitter.im/FreeCodeCamp/YouTube) for broadcasting our monthly summits and pair programming as well as uploading tutorials and hikes.
```
Comment (abhisekp) [inline Twitch.md]: I think we should give the channel here and link to the chat room below i.e.
> `...focus onto using [YouTube](https://www.youtube.com/freecodecamp) for broadcasting ...`

File: Twitch.md [hunk 10]
```diff
@@ -1,7 +1,17 @@
-FreeCodeCamp uses Twitch.tv to broadcast live streams. This can be anything from live pair programming to our monthly Summit, where big news events and new developments are announced.
+### Twitch is a popular live streaming service that Free Code Camp moved away from
```
Comment (abhisekp) [inline Twitch.md]: *Just one tiny little improvement.*
Use Heading 4 for consistency.

----
I'm going to merge this in 5 mins

**Comment by abhisekp** [comment]
@theflametrooper will you squash all the commits? or I can squash and merge the PR.

**Comment by abhisekp** [comment]
# :+1: LGTM
Thanks. :tada: :smiley:

**Pull request merged**